src/core/misc.c

```diff
@@ -741,15 +741,16 @@ int expand_escape(const char **data)
 	}
 }
 
-/* Escape all '"', "'" and '\' chars with '\' */
-char *escape_string(const char *str)
+/* Escape all the characters in `what' with a backslash */
+char *escape_string(const char *str, const char *what)
```

Comment:
This looks API-breaking (unless you're 200% certain no modules use this)

Reply:
yes, I resurrected this from https://github.com/irssi/irssi/pull/658/commits/027acffb4208d7e6ba8e229cbf6c3dae6f5dabaf

should we add another escape function instead?

Reply:
Sure, can't imagine any other way.